src/helpers/color.ts

```diff
@@ -274,6 +283,26 @@ export function rgbaToHSLA(rgba: RGBA): HSLA {
   return { a: rgba.a, h, s, l };
 }
 
-export function isSameColor(color1: Color, color2: Color): boolean {
-  return isColorValid(color1) && isColorValid(color2) && toHex(color1) === toHex(color2);
+export function hslaToHex(hsla: HSLA): Color {
+  return rgbaToHex(hslaToRGBA(hsla));
+}
+
+export function hexToHSLA(hex: Color): HSLA {
+  return rgbaToHSLA(colorToRGBA(hex));
+}
+
+export function isSameColor(color1: Color, color2: Color, tolerance: number = 0): boolean {
```

Comment:
It may be overkill here, but there are some "scientifically validated" way to compute the distance between two colors as well as the tolerance to consider two colors to be identical : 

https://en.wikipedia.org/wiki/Color_difference#Tolerance

Reply:
Overkill indeed I think.
We do not build a drawing software ^^

Reply:
yup, we discussed this with Anthony afterwards and since the idea here is to have a 'conversion' tolerance rather than a 'human perception' tolerance, the  distance seems ok.